lib/firebase.dart

```diff
@@ -72,3 +74,32 @@ void _fbAuthChanged(
 
   loginCallback();
 }
+
+// Read data
+Future<model.Config> fetchConfig() async {
+  logger.debug('Fetching config from firebase ..');
+  var chartsConfigRef = firebase.firestore().doc(fb_constants.metadataPath);
+  var configSnap = await chartsConfigRef.get();
```

Comment:
nit: prefer to use the full word for less used terms: `configSnap` -> `configSnapshot`

Reply:
Fixed